### ISSUE TITLE
fpGUI widgetset for Lazarus

### ISSUE THREAD

**Issue by Alexey-T** [graemeg/fpGUI#116]
Hello. Is it possible to compile the GUI Lazarus app with the fpGUI widgetset, or some issues are still here? Please.

**Comment by graemeg** [comment]
I'm not the maintainer of the Lazarus "LCL-fpGUI" widgetset, though I have contributed quite a lot of the functionality - in the hopes that someone else would take that over.  My last contribution [if I recall correctly] was to finally implement correct rendering of TLabel wiget. That was the last widget from the first three component tabs that caused a basic LCL-fpGUI app to crash at startup.

If your question was more towards "Can I compile Lazarus IDE itself with LCL-fpGUI".... Well, that I've never tried. The IDE itself uses a lot of custom widgets, so not sure if all the Win32 API methods have been implemented in the LCL-fpGUI widgetset. And yes, LCL is based on Win32, so every LCL widgetset needs to implement (emulate) some of the Win32 drawing APIs.

You might get a better answer if you ask in the Lazarus mailing list, as [this repo](https://github.com/graemeg/fpGUI/i) is only about fpGUI, not LCL-fpGUI.
